save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -187,6 +187,7 @@ class DockerService(private val configProperties: ConfigProperties) {
                     |RUN ln -fs /usr/share/zoneinfo/UTC /etc/localtime
                     |RUN chmod +x $executionDir/$SAVE_AGENT_EXECUTABLE_NAME
                     |RUN chmod +x $executionDir/$SAVE_CLI_EXECUTABLE_NAME
+                    |RUN chmod +x $executionDir/diktat-rules/src/test/resources/test/smoke/ktlint
```

Comment:
Pls change it to safe call (e.g. `if [ -f <path> ]` or `find -exec`), so that if there is no ktlint file, build doesn't fail. Also, pls add a comment that this is a temporary workaround link to #277 . This way we will be able to run diktat tests while #277 is still WIP.